### PR TITLE
added a new html file

### PR DESCRIPTION
**Pull request by ashna-53**

### CODE REVIEW

File: homepage.html [hunk 1]
```diff
@@ -0,0 +1,112 @@
...
+        <li class="item">
+          <a href="#" class="nav_link">
+            <span class="navlink_icon">
+              <i class="bx bxs-magic-wand"></i>
+            </span>
+            <span class="navlink" >Logos</span>
+            
+          </a>
+          
+          <!-- <div class="col custom-height-1 custom-shape-col ">
```
Comment (ChristoBose) [inline homepage.html]: Remove Unnecessary code-comments

File: homepage.html [hunk 2]
```diff
@@ -0,0 +1,112 @@
...
+            <img src="shapes\love_small_black.png"onclick="changeTshirtImage('shapes/love_small_black.png')">
+          </div> -->
+
+          <!-- <div class="col custom-height-1 custom-shape-col ">
+            <img src="shapes\love_small_black.png"onclick="changeTshirtImage('shapes/love_small_black.png')">
+          </div> -->
+          <!-- <div class="col custom-height-1 custom-shape-col ">
+            <img src="shapes\love_small_black.png"onclick="changeTshirtImage('shapes/love_small_black.png')">
+          </div> -->
```
Comment (ChristoBose) [inline homepage.html]: Remove Unnecessary comments.

File: homepage.html (hunk 1 above)
Comment (ashna-53) [reply homepage.html]: I have removed unneccessary comments from the code . Please check it

File: homepage.html (hunk 2 above)
Comment (ashna-53) [reply homepage.html]: I have made the changes .Please check it again

Comment (ChristoBose) [reply homepage.html]: okay.Thankyou

File: homepage.html (hunk 1 above)
Comment (ChristoBose) [reply homepage.html]: Okay .Thankyou.

**Comment by ChristoBose** [comment]
Well done

**Pull request merged**